.vscode/launch.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
could you please remove this file from here?

Reply:
Yes, of course